Source: https://bugs.webkit.org/show_bug.cgi?id=254286

Usage initial-exec TLS is incompatible with dlopen() on musl.

See also: https://gitlab.freedesktop.org/mesa/mesa/-/commit/8570a2a280587a1e43ac11ad46ad62dfdd6c7b39
 .../platform/graphics/gbm/GraphicsContextGLANGLELinux.cpp   | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)
--- a/Source/WebCore/platform/graphics/gbm/GraphicsContextGLANGLELinux.cpp	
+++ a/Source/WebCore/platform/graphics/gbm/GraphicsContextGLANGLELinux.cpp	
@@ -108,7 +108,11 @@ RefPtr<PixelBuffer> GraphicsContextGLANGLE::readCompositedResults()
 
 bool GraphicsContextGLANGLE::makeContextCurrent()
 {
-    static thread_local TLS_MODEL_INITIAL_EXEC GraphicsContextGLANGLE* s_currentContext { nullptr };
+    #ifdef __GLIBC__
+        static thread_local TLS_MODEL_INITIAL_EXEC GraphicsContextGLANGLE* s_currentContext { nullptr };
+    #else
+        static thread_local GraphicsContextGLANGLE* s_currentContext { nullptr };
+    #endif
 
     if (s_currentContext == this)
         return true;
-- 
